src/views/dashboard/data-explorer/components/log.vue

```diff
@@ -2,25 +2,25 @@
 a-list(:hoverable="true" size="small")
   template(#header)
     | {{$t('dataExplorer.logs')}}
-  a-list-item(v-for="item of logListData" :key="item")
-    a-tooltip(:content="item.error ? item.error : item.runCode")
+  a-list-item(v-for="item of logs" :key="item")
+    a-tooltip(:content="item.error ? item.error : item.sql")
```

Comment:
Are we sure we want to change 'code' to 'sql'? What if our editor can support other languages such as python in the future?

Reply:
> 

I think pyhon has a lot of lines of code and should display logs in a different component than SQL requests